x/group/server/testsuite/suite.go

```diff
@@ -793,6 +793,323 @@ func (s *IntegrationTestSuite) TestCreateGroupAccount() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestUpdateGroupAccountAdmin() {
+	admin, newAdmin := s.addr1, s.addr2
+	groupRes, err := s.msgClient.CreateGroup(s.ctx, &group.MsgCreateGroupRequest{
+		Admin:    s.addr1.String(),
+		Members:  nil,
+		Metadata: nil,
+	})
+
+	s.Require().NoError(err)
+	myGroupID := groupRes.GroupId
+	groupAccount := &group.MsgCreateGroupAccountRequest{
+		Admin:    admin.String(),
+		Metadata: nil,
+		GroupId:  myGroupID,
+	}
+
+	err = groupAccount.SetDecisionPolicy(group.NewThresholdDecisionPolicy(
+		"1",
+		gogotypes.Duration{Seconds: 1},
+	))
+	s.Require().NoError(err)
+
+	groupAccountRes, err := s.msgClient.CreateGroupAccount(s.ctx, groupAccount)
+	s.Require().NoError(err)
+	groupAccountAddress := groupAccountRes.GroupAccount
+
+	specs := map[string]struct {
+		req             *group.MsgUpdateGroupAccountAdminRequest
+		policy          group.DecisionPolicy
+		expGroupAccount *group.GroupAccountInfo
+		expErr          bool
+	}{
+		"with wrong admin": {
+			req: &group.MsgUpdateGroupAccountAdminRequest{
+				Admin:        s.addr5.String(),
+				GroupAccount: groupAccountAddress,
+				NewAdmin:     newAdmin.String(),
+			},
+			policy: group.NewThresholdDecisionPolicy(
```

Comment:
`policy` is not really needed here since it's always the same and we're testing admin update, let's just use one variable instead.